dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckerTest.java

```diff
@@ -122,7 +148,7 @@ public void testFindInvalidReferences_selfReferenceFromAbstractClassToInterface(
     SymbolReferenceSet symbolReferenceSet = ClassDumper.scanSymbolReferencesInJar(httpClientJar);
 
     JarLinkageReport jarLinkageReport =
-        staticLinkageChecker.generateLinkageReport(httpClientJar, symbolReferenceSet);
+        staticLinkageChecker.generateLinkageReport(httpClientJar, symbolReferenceSet, null);
```

Comment:
I don't think `generateLinkageReport()` should accept null. Instead pass an empty list or something.

Reply:
done